Sources/AWSLambdaRuntimeCore/LambdaRunner.swift

```diff
@@ -33,7 +33,7 @@ internal final class LambdaRunner {
     /// Run the user provided initializer. This *must* only be called once.
     ///
     /// - Returns: An `EventLoopFuture<LambdaHandler>` fulfilled with the outcome of the initialization.
-    func initialize<Handler: ByteBufferLambdaHandler>(logger: Logger, terminator: LambdaTerminator, handlerType: Handler.Type) -> EventLoopFuture<Handler> {
+    func initialize(handlerType: (some ByteBufferLambdaHandler).Type, logger: Logger, terminator: LambdaTerminator) -> EventLoopFuture<any ByteBufferLambdaHandler> {
```

Comment:
We must enforce 5.7 if we want to use this syntax. Currently we require 5.6. Also IMO we def. should not return `any` here. We should preserve the type.

Reply:
this PR already changes the requirement to 5.7, unless I missed something?

Reply:
It only changes this for all the examples. It also drops the 5.4 and 5.5 Package.swift. But the Package.swift without version only requires 5.6.

Reply:
Also I think you are required to use the old generic syntax here, if we don't want to drop the types. Since we need create a connection between the input and the output type.

Reply:
> But the Package.swift without version only requires 5.6.

good catch

Reply:
> Also I think you are required to use the old generic syntax here, if we don't want to drop the types. Since we need create a connection between the input and the output type.

I don't think this can work with the old syntax. the Handler type is stored in LambdaRuntime which needs to loose it old generic signature, so they do not align any more